rlinf/data/io_struct.py

```diff
@@ -302,6 +319,21 @@ def merge_result_list(
                 if merged_result.rollout_logprobs is None:
                     merged_result.rollout_logprobs = []
                 merged_result.rollout_logprobs.extend(res.rollout_logprobs)
+
+        # TODO Align advantages and rewards format
+        if (
+            isinstance(merged_result.advantages, list)
+            and len(merged_result.advantages) > 0
+            and isinstance(merged_result.advantages[0], torch.Tensor)
+        ):
+            merged_result.advantages = torch.cat(merged_result.advantages, dim=0)
```

Comment:
should we add `else`? if `merged_result.advantages[0]` is not tensor, then we should raise or ignore?

Reply:
OK